### ISSUE TITLE
[BUG] Fix introduced in #60 breaks backwards compatibility with old node.js versions

### ISSUE THREAD

**Issue by pniewiejski** [npm/hosted-git-info#65]
# What / Why
Changes introduced in #60 (specifically in commit: https://github.com/npm/hosted-git-info/commit/f2cdfcf33ad2bd3bd1acdba0326281089f53c5b1) break compatibility for older versions of node.js which do not support `const` & `let` variable declarations.   Notice that in `v2.8.7`, apart from one place in tests, it is the only place where `const` (or `let` for that matter) is used in the whole project. 

So before #60 this project was backwards compatible and after #60 it is not. I think this should be considered a breaking change. 

This fix (#60) was introduced between v2.8.6 and v2.8.7. Introducing breaking changes and bumping only the _patch_ number is IMHO a practice that should be avoided. 

## Where / affected versions  
Affected version is `v2.8.7`. 
node.js version: `0.10.*` (Yes, it's still there... 😢)

## Expected outcome

Change `const` declaration to `var`.
```js
// index.js +123
const authmatch = giturl.match(/[^@]+@[^:/]+/)
```

**Comment by ljharb** [comment]
This is breaking me as well for a number of packages https://travis-ci.com/inspect-js/has-template-literals/jobs/292483820

v2 can't use `const` or `let`, in order to maintain compatibility. Can v2.8.7 be reverted, or updated to only use `var`?

**Comment by darcyclarke** [comment]
@ljharb @pniewiejski we'll revert this to use `var` & ship a patch either before or on this next Tuesday

**Comment by ljharb** [comment]
Thanks! It’d be great to do it sooner, since all my packages’ CI is failing in the meantime unless i add a commit pegging the version of hosted-git-info. Perhaps you could make “latest” be 2.8.6 right now, so that people don’t have to get 2.8.7 unless they ask for it?

**Comment by darcyclarke** [comment]
This was fixed in https://github.com/npm/hosted-git-info/pull/66 & is landed as [`hosted-git-info@2.8.8`](https://www.npmjs.com/package/hosted-git-info?activeTab=versions). We'll try to roll out a cli update as soon as possible which will include this dep update (likely as `npm@6.14.2`)
